charts/garnet/README.md

```diff
@@ -10,6 +10,23 @@ A Helm chart for Microsoft garnet
 
 * <https://github.com/microsoft/garnet.git>
 
+## Usage
+
+[Helm](https://helm.sh) must be installed to use the charts. Please refer to
+Helm's [documentation](https://helm.sh/docs) to get started.
+
```

Comment:
Is there a way users can provide their own `garnet.conf` configuration file for use with the helm chart?

Reply:
Actually no in this helm chart but this could be adressed by [this PR](https://github.com/microsoft/garnet/pull/531).

Reply:
OK would be great to see how to port this into your helm chart in a separate PR, as the other PR seems to create a duplicate.